data/brands/amenity/restaurant.json

```
@@ -43,6 +43,16 @@
     }
   },
   "items": [
+   {
+      "displayName": "Pokawa",
+      "locationSet": {"include": ["fr","lu","be","ch","es","pt"]},
+      "tags": {
+        "amenity": "restaurant",
+        "brand": "Pokawa",
+        "brand:wikidata": "Q123018553",
```

```suggestion
        "brand:wikidata": "Q123018553",
        "cuisine": "poke",
```
Based on https://pokawa.com/fr_fr/notre-concept/ , this looks like the best value for `cuisine` to me.